testsuite/cluster-test/src/main.rs

```diff
@@ -379,6 +379,22 @@ impl ClusterUtil {
                 "Deploying with {} tag for validators and fullnodes",
                 image_tag
             );
+            let asg_name = format!(
+                "{}-k8s-testnet-validators",
+                cluster_swarm
+                    .get_workspace()
+                    .await
+                    .expect("Failed to get workspace")
+            );
+            aws::set_asg_size(
+                (args.k8s_num_validators
+                    + (args.k8s_fullnodes_per_validator * args.k8s_num_validators))
+                    as i64,
+                3,
```

Comment:
LBT uses 30-node config, and CP uses 100. 3 seems plenty for LBT but not CP.  Should we make the buffer % based instead?

Reply:
I think CP also uses 30-node config. But either way, percent is a good idea. Updated to use `buffer_percent`

Reply:
Yep, all tests are using 30 validator + 30 full nodes setup